crates/factor-outbound-http/src/lib.rs

```diff
@@ -81,6 +82,8 @@ pub struct InstanceState {
     component_tls_configs: ComponentTlsConfigs,
     self_request_origin: Option<SelfRequestOrigin>,
     request_interceptor: Option<Box<dyn OutboundHttpInterceptor>>,
+    // Connection-pooling client for 'fermyon:spin/http' interface
```

Comment:
Nit: make this visible to IDEs 

```suggestion
    /// Connection-pooling client for 'fermyon:spin/http' interface
```